lib/scholar/integrate/trapezoidal.ex

```diff
@@ -0,0 +1,182 @@
+defmodule Scholar.Integrate.Trapezoidal do
```

Comment:
Do we want to have one module per integration or a single `Scholar.Integrate` module with all integration methods?

Reply:
I believe it's best to have one module with the different methods. We could go so far as to have a single function that delegates to different implementations based on an option

Reply:
In general, the choice for one integration method over the other depends on the type of function you're integrating.

For instance, in circuit simulators it is common to have a drop-down menu in which you choose  the integration method. Then, if one gives artifacts, another might work better.

Reply:
I almost implemented the Simpson method in a separate file.

Reply:
I'm ok with having private separate files for them, but I really think that it would be best to have the public interface more tightly bound